try:
    print(8/0)
except Exception as e:
    print(e)